livekit-agents/livekit/agents/worker.py

```diff
@@ -485,6 +485,8 @@ async def _recv_task():
                     self._handle_availability(msg.availability)
                 elif which == "assignment":
                     self._handle_assignment(msg.assignment)
+                elif which == "termination":
+                    await self._handle_termination(msg.termination)
```

Comment:
This needs to be in a background task (the process can take 60s to shutdown)

Put it inside self._tasks